src/studio/src/designer/backend/Services/Implementation/RepositorySI.cs

```diff
@@ -1035,7 +1035,7 @@ public bool CreateModel(string org, string app, ModelMetadata modelMetadata, XDo
         }
 
         /// <summary>
-        ///  This logic is limited to having one datamodel per app. Needs to be updated for expanded functionality
+        ///  Updates application model
```

Comment:
Slightly more descriptive comment here please :) 
"Updates application with new app logic model" or something like it maybe?

Reply:
Sure, what about the properties TaskId, AllowedContentTypes, MaxCount and MinCount?

Reply:
Good question. Seems like maxCount/minCount and allowedContentTypes props are set to the same values as the default-element that is in the template, so I think that should be ok for now. I'm less sure about taskId. @TheTechArch Do you have any input here?

Reply:
My suggestion for taskId is this:
- check if there already exists another dataType with appLogic.
  - if yes: omit the taskID when adding a new dataType
  - if no: set the taskId to the default task "Task_1" as we do now.
The app developer will then have to manually update the taskId if they have more than 1 data tasks. We should document this well, until we have a more clear design for how one navigates between tasks/datamodels in studio.

Reply:
done